### PR TITLE
Prevent sidebar and fullscreen icons from overlapping with the scrollbar (issue 1765)

### PR DESCRIPTION
**Pull request by ma12-co**
Fix https://github.com/nextcloud/spreed/issues/1765

### CODE REVIEW

File: css/style.scss [hunk 1]
```diff
@@ -346,7 +346,7 @@ input[type="password"] {
 #app-sidebar-trigger {
 	background-color: transparent;
 	border: none;
-	margin: 0;
+	margin: 0 10px 0px 0px;
```
Comment (nickvergessen) [inline css/style.scss]:
```suggestion
	margin: 0 10px 0 0;
```

**Comment by ma12-co** [comment]
I agree, it is hacky :) 
for the future Vue implementation there could be a vertical bar component alongside the sidebar that hosts those three buttons maybe?

**Comment by danxuliu** [comment]
> for the future Vue implementation there could be a vertical bar component alongside the sidebar that hosts those three buttons maybe?

It should not, because the share button belongs to the new message form, it is independent from the other two buttons even if it would be nice that the three are aligned. Anyway, we will think about that when the time comes ;-)

**Comment by jancborchardt** [comment]
@ma12-co thanks a lot for the design fix! Cool to have you in the team :rocket:

**Comment by ma12-co** [comment]
Thanks @jancborchardt , I'm thrilled to be onboard :)

**Pull request merged**